internal/app/feed/feed_group_user.go

```diff
@@ -68,9 +68,9 @@ func (f *GroupUserFeed) Export(ctx context.Context, apiClient *api.Client, expor
 	logger.Infof("%s: exporting for org_id: %s", feedName, orgID)
 
 	exporter.InitFeed(f, &InitFeedOptions{
-		// Clear this table before loading data.
-		// This table does not receive updates, it is only refreshed.
-		Truncate: true,
+		// Truncate files if upserts aren't supported.
+		// This ensure that the export does not contain duplicate rows
+		Truncate: !exporter.SupportsUpsert(),
```

Comment:
We actually intentionally truncate this table on every run. As if we don't users who are removed from groups will remain. 

Can you add functionality to delete rows based on the organisation ID? That way it only recreates data for that specific organisation.